src/main/java/com/example/employeecrudapplication/validator/EmployeeValidator.java

```diff
@@ -0,0 +1,45 @@
+package com.example.employeecrudapplication.validator;
+
+import com.example.employeecrudapplication.exception.EmployeeValidationException;
+import com.example.employeecrudapplication.model.EmployeeDto;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmployeeValidator {
+
+    public void validateEmail(String email) {
+        if (
+                StringUtils.isBlank(email)
+                || (!email.contains("@"))
+                || (!email.contains("."))
+        ) {
+            throw new EmployeeValidationException("Employee Email is not present in the request");
+        }
+    }
+
+    public void validateFields(EmployeeDto employee) {
+        if (employee == null) {
+            throw new EmployeeValidationException("Null Pointer Exception! Employee request is null");
```

Comment:
For EmployeeValidationException make response status 400 BAD REQUEST

Reply:
Tried doing it like this, is this correct?
![image](https://user-images.githubusercontent.com/100875409/219565397-45fc211e-02e4-44a1-ac41-290368193da3.png)

Reply:
@larinpaul https://www.baeldung.com/spring-response-status-exception

Reply:
I don't understand how to make a solution for this. This issue will be reserved for a Skype call